src/components/v5/shared/UserPopover/partials/UserInfo.tsx

```diff
@@ -86,16 +86,16 @@ const UserInfo: FC<UserInfoProps> = ({
       )}
       {domains?.length ? (
         <div
-          className={clsx({
-            'px-6 pb-6 pt-2': isTopContributorType,
+          className={clsx('pt-6', {
+            'px-6 pb-6': isTopContributorType,
           })}
```

Comment:
What do you think about this notation?
```
className={clsx({
  'p-6': isTopContributorType,
  'pt-6': !isTopContributorType,
})}
```

Reply:
It's outdated after my last changes. Could you check this again? :)